### PR TITLE
Highlight Trending link correctly

### PR DESCRIPTION
**Pull request by busches**
Highlight Trending link when on the Trending page and subpages and
remove highlighting from Explore when on Trending pages.

Fixes issue #705

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -118,12 +118,19 @@ function moveMarketplaceLinkToProfileDropdown() {
...
 	issuesLink.parentNode.after(
 		<li class="header-nav-item">
-			<a href="/trending" class="js-selected-navigation-item HeaderNavlink header-nav-link px-2" data-hotkey="g t">Trending</a>
+			<a href="/trending" class={`js-selected-navigation-item HeaderNavlink header-nav-link px-2 ${selectedClass}`} data-hotkey="g t">Trending</a>
 		</li>
 	);
+
+	// Explore link highlights /trending urls by default, remove that behavior
+	if (pageDetect.isTrending()) {
+		const exploreLink = await safeElementReady('a[href="/explore"]');
```
Comment (bfred-it) [inline src/content.js]: Can you make this selector more specific?

Comment (busches) [reply src/content.js]: Something like this? `safeElementReady('.HeaderMenu a[href="/explore"], .header-menu a[href="/explore"])`. Guessing on the `header-menu` for GHE, but it'll probably be different there.

Comment (bfred-it) [reply src/content.js]: Goes GHE also have Trending? :|

Also yeah let’s check the necessary classes for GHE first so you don’t have to send another PR next week 😃

Comment (busches) [reply src/content.js]: Yes, it has both options actually. GHE is easier with `'a.header-nav-link[href="/explore"]'`. I'll update the PR.

Comment (bfred-it) [reply src/content.js]: `.catch(() => null)`

**Comment by busches** [comment]
@bfred-it added in the catch, forgot you could treat them like a chained promise, thanks!

**Comment by bfred-it** [comment]
But you'll still need an `if` otherwise it'll be `null.classList.remove(...)`

**Comment by busches** [comment]
Fixed.

**Comment by bfred-it** [comment]
Thanks @busches!

**Pull request merged**